README.md

```diff
@@ -1,88 +1,46 @@
-- We've built a handy debugging/validation dashboard for you to quickly visualize the data in your subgraph. It's deployed to [subgraphs.xyz](https://subgraphs.xyz/) and the source code is under `dashboard` if you want to spin it up locally.
-- There is also a cli-tool called `messari-cli`. It utilized The Graph CLI and exploits the standard directory structure to template and prepare subgraph deployments for you with simple commands. You can learn more about installing and using the CLI by looking at the README.md within the `messari-cli/docs` folder.
-- Verify your subgraph against other sources and include specific links to these sources in the README. Below are some common sources:
-  - Project's official analytics dashboard
-  - [DeFi Llama](https://defillama.com/) (for TVL)
-  - [Dune Analytics](https://dune.xyz/)
-  - [TokenTerminal](https://www.tokenterminal.com/terminal)
+Go to [`docs/SETUP.md`](./docs/SETUP.md) to learn how to setup your machine for Messari subgraph development.
 
-> For other contributing guidelines, please refer to [Contributing.md](./docs/Contributing.md)
+## Learn the Project
 
-## Video Walkthrough Series of Subgraph Development Workflow for Messari
+It is important to familiarize yourself with the project structure and tooling to build efficiently. Go to [`docs/STRUCTURE.md`](./docs/STRUCTURE.md) and [`docs/TOOLING.md`](./docs/TOOLING.md) to learn more.
 
-**CLI Installation:**
+- Familiarize yourself with our schemas labeled `schema-{protocol type}.graphql`. Read more details in [`docs/SCHEMA.md`](./docs/SCHEMA.md)
+- We update our schemas as necessary. You can find out about each upgrade in [`docs/CHANGES.md`](./docs/CHANGES.md)
 
-- `npm install -g messari-subgraph-cli`
-- `npm install -g mustache`
+## Becoming a Subgraph Developer
 
-**Deployment.json Walkthough:**
+Becoming a good subgraph developer will take patience and practice. The following resources are for developers of all skill levels to learn the ins and outs of subgraph development. 👾
 
-[![Deployment.json Walkthrough](https://img.youtube.com/vi/cgCNkRmM4NM/0.jpg)](https://youtu.be/cgCNkRmM4NM)
+- For a full walkthrough of our subgraph development process visit [`docs/WALKTHROUGH.md`](./docs/WALKTHROUGH.md).
+- Resources for development of varying levels can be found in [`docs/RESOURCES.md`](./docs/RESOURCES.md).
+- To learn about common errors, best error handling practices, and debugging see [`docs/ERRORS.md`](./docs/ERRORS.md)
+- Subgraph performance is also a concern. Learn about indexing / querying performance by reading [`docs/PERFORMANCE`](./docs/PERFORMANCE.md)
+- To learn about Messari standard methodologies see [`docs/METHODOLOGY.md`](./docs/METHODOLOGY.md)
```

Comment:
Does it make sense to add a mention to methodologies outside of the `becoming a subgraph developer` section? For whomever is interested in knowing about that but is not a developer nor wants to be